zerver/actions/invites.py

```diff
@@ -36,10 +36,13 @@
 from zerver.tornado.django_api import send_event
 
 
-def notify_invites_changed(realm: Realm) -> None:
+def notify_invites_changed(realm: Realm, *, acting_user: Optional[UserProfile] = None) -> None:
     event = dict(type="invites_changed")
     admin_ids = [user.id for user in realm.get_admin_users_and_bots()]
-    send_event(realm, event, admin_ids)
+    recipient_ids = admin_ids
+    if acting_user:
+        recipient_ids.append(acting_user.id)
+    send_event(realm, event, recipient_ids)
```

Comment:
I am not completely sure, but we might need a changelog entry for this too. @laurynmm FYI.

Reply:
Yeah, looking at https://zulip.com/api/get-events#invites_changed, we would need to update the event description for the new users that get these events as well as make a **Changes** note for the previous state (only admins received these events) and a corresponding entry/bullet point in the API changelog for feature level 202 (at this point).